components/brave_rewards/resources/shared/components/newtab/rewards_card.style.ts

```diff
@@ -110,15 +111,34 @@ export const disconnectedArrow = styled.div`
 
 export const balance = styled.div`
   margin-top: 16px;
+  margin-bottom: 16px;
   font-size: 12px;
   line-height: 18px;
+
+  &.flat {
+    margin-bottom: 6px;
+  }
 `
 
 export const balanceTitle = styled.div`
   color: rgba(255, 255, 255, 0.8);
   letter-spacing: 0.01em;
 `
 
+export const balanceSpinner = styled.div`
+  .icon {
+    height: 36px;
+    vertical-align: middle;
+    margin-right: 4px;
+  }
+
+  .loading {
```

Comment:
Can these styles just be applied to the container element (`balanceSpinner`) so that we don't need this `span`?

Reply:
Resolving this as per our Slack convo.